src/main/java/com/spotify/reaper/ReaperApplication.java

```diff
@@ -172,7 +195,27 @@ public void run(ReaperApplicationConfiguration config,
     SchedulingManager.start(context);
 
     LOG.info("resuming pending repair runs");
-    context.repairManager.resumeRunningRepairRuns(context);
+    
+    if (context.storage.getStorageType() == StorageType.CASSANDRA) {
+      // Allowing multiple Reaper instances to work concurrently requires
+      // us to poll the database for running repairs regularly
+      // only with Cassandra storage
+      scheduler.scheduleWithFixedDelay(
+        new Runnable() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1604)